README.md

```diff
@@ -1,4 +1,11 @@
 # spec
-This repository houses the MODIP Specification.
+This repository houses various specifications within MODIP.
 
-**This is currently a draft.** 
+## `index_spec`
+This is the MODIP Index Specification. This is a complex specification which may be used by a MODIP-hosted mod Index, but anyone is allowed to use it for their own purposes.
```

Comment:
So many extra words. I feel like you don't really have a vision for what this does.

Reply:
Yeah, I just wanted to specify that it's not exclusive for the MODIP-hosted Index. But I will reword it